packages/reports/src/Containers/progress-slider-stream.tsx

```diff
@@ -1,11 +1,19 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import { ProgressSlider } from '@deriv/components';
 import { getCurrentTick } from '@deriv/shared';
 import { connect } from 'Stores/connect';
 import { getCardLabels } from '_common/contract';
+import moment from 'moment';
+import { TContractInfo } from '@deriv/shared/src/utils/contract/contract-types';
```

Comment:
export this file in the `packages/shared/src/utils/contract/index.ts` path so you can use export it like regular files from shared
```suggestion
import { TContractInfo } from '@deriv/shared';
```